stripe-core/src/main/java/com/stripe/android/core/networking/ConnectionFactory.kt

```diff
@@ -48,24 +52,31 @@ interface ConnectionFactory {
         }
 
         private fun openConnectionAndApplyFields(request: StripeRequest): HttpURLConnection {
-            return (URL(request.url).openConnection() as HttpURLConnection).apply {
-                testConnectionCustomization?.invoke(this)
+            val actualRequest = if (BuildConfig.DEBUG) {
```

Comment:
I added these `if (BuildConfig.DEBUG)` checks so that it could be compiled out when we do our release builds to merchants.